### PR TITLE
VDS-2923 Implement the physical products exporting

### PR DESCRIPTION
**Pull request by trueboroda**
Description: 

--

QA-test: VDS-2926

Demo-test: VDS-2927

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.CatalogExportImport_1.0.0-pr-17-d8e28b93.zip

### CODE REVIEW

File: src/VirtoCommerce.CatalogExportImportModule.Core/Models/CsvPhysicalProduct.cs [hunk 1]
```diff
@@ -116,5 +117,36 @@ public sealed class CsvPhysicalProduct : IImportable, IExportable
...
+            Visible = product.IsActive;
+            TrackInventory = product.TrackInventory;
+            PackageType = product.PackageType;
+            MaxQuantity = product.MaxQuantity;
+            MinQuantity = product.MinQuantity;
+            ManufacturerPartNumber = product.ManufacturerPartNumber;
+            MeasureUnit = product.MeasureUnit;
+            WeightUnit = product.WeightUnit;
+            Weight = product.Weight;
+            Lenght = product.Length;
```
Comment (Pav-el) [inline src/VirtoCommerce.CatalogExportImportModule.Core/Models/CsvPhysicalProduct.cs]: LengTH?

File: src/VirtoCommerce.CatalogExportImportModule.Core/ModuleConstants.cs [hunk 2]
```diff
@@ -5,6 +5,30 @@ namespace VirtoCommerce.CatalogExportImportModule.Core
...
+
+            public const string PhysicalProduct = "PhysicalProduct";
+        }
+
+        public static class PushNotificationsTitles
+        {
+            public static readonly IReadOnlyDictionary<string, string> Export = new Dictionary<string, string>()
+            {
+                {DataTypes.EditorialReview, "Product descriptions export"},
+                {DataTypes.PhysicalProduct, "Physical product export"},
```
Comment (Pav-el) [inline src/VirtoCommerce.CatalogExportImportModule.Core/ModuleConstants.cs]: productS?

File: src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/dialogs/exportConfirm-dialog.tpl.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<div class="modal-header catalog-export-import">
+    <h3 class="modal-title">{{ 'catalogExportImport.dialogs.'+ dataType + '-export.title' | translate }}</h3>
+</div>
+<div class="modal-body" ng-switch on="dataType">
```
Comment (Pav-el) [inline src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/dialogs/exportConfirm-dialog.tpl.html]: ng-switch seems not working here

**Review by Pav-el — APPROVED** [review]

File: src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/module.js [hunk 4]
```diff
@@ -33,7 +43,7 @@ var module = angular.module(moduleName, ['ui.grid.autoFitColumns']).run(['virtoC
             icon: 'fas fa-box',
             controller: 'virtoCommerce.catalogExportImportModule.fileUploadController',
             template: 'Modules/$(VirtoCommerce.CatalogExportImport)/Scripts/blades/file-upload.tpl.html',
-            predefinedDataType: { key: 'Physical Products', value: 'CatalogProduct' }
+            predefinedDataType: { key: 'Physical Products', value: 'PhysicalProduct' }
```
Comment (ilyawzrd) [inline src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/module.js:46]: Why did you rename this value? It should stay 'CatalogProduct'

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

**Review by ilyawzrd — APPROVED** [review]

**Pull request merged**